src/styles/ForecastSummary.css

```diff
@@ -0,0 +1,65 @@
+.forecast-summary {
```

Comment:
There are a lot being repeated here:
This would do the same tricks:
`````
.forecast-summary {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  padding: 20px;
  margin: 0 auto 1rem;
  width: 250px;
  height: 200px;
  background: #fff;
  border: 5px solid #000;
  border-radius: 50%;
  box-shadow: inset -10px 10px 0 10px #ccc;
}

`````